### DISCUSSION TITLE
the MAUI dependencies are not restoring in solution

### DISCUSSION THREAD

**Discussion by gdayal80** [dotnet/maui#9892] [Q&A]
the MAUI dependencies are not restoring in solution

I am using visual studio 2022 v 17.3.3 on windows 10 x64

![maui dependencies](https://user-images.githubusercontent.com/37060786/188277996-4ece8603-5dec-464f-adf1-bff32f7d79fa.jpg)

**Comment by eldendiss** [comment]
Same, looks like there was some update as maui WorkloadManifest requires all packs to be at version 6.0.532. However nuget.org feed doesnt hold such versions. Really annoying, as I cant even build the solution.
```
dotnet workload restore
Installing workloads: maui-tizen

Installing pack Microsoft.Maui.Core.Ref.tizen version 6.0.532...
Workload installation failed. Rolling back installed packs...
Rolling back pack Microsoft.Maui.Core.Ref.tizen installation...
Workload installation failed: microsoft.maui.core.ref.tizen::6.0.532 is not found in NuGet feeds https://api.nuget.org/v3/index.json".
```

EDIT: I even tried reinstalling VS with no success.
I managed to get it working by building latest release 6.0.424, replacing the one provided by VS installer.

**Reply by gdayal80** [reply]
ok I'll try your solution.

Thnx

**Reply by gdayal80** [reply]
yes it worked, thanks
